aya-log/src/lib.rs

```diff
@@ -324,8 +327,13 @@ impl Log for DefaultLogger {
     }
 }
 
+const MAP_NAME: &str = "AYA_LOGS";
```

Comment:
Please move that const to the top of the file.

Reply:
Done.